presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestDatabaseShardManager.java

```diff
@@ -747,4 +784,14 @@ private static Node createTestingNode()
     {
         return new PrestoNode(UUID.randomUUID().toString(), URI.create("http://test"), NodeVersion.UNKNOWN, false);
     }
+
+    private int columnCount(long tableId)
+            throws SQLException
+    {
+        try (Statement st = dummyHandle.getConnection().createStatement()) {
+            try (ResultSet rs = st.executeQuery(format("SELECT * FROM %s LIMIT 0", shardIndexTable(tableId).toUpperCase(ENGLISH)))) {
```

Comment:
No need to uppercase here, since this is a case-insensitive identifier in a SQL query